PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -1,21 +1,66 @@
 from aloe import *
-from iota import Iota
+from iota import Iota, ProposedTransaction, Address
 from iota.commands.core import add_neighbors
 from util import static_vals
+from util.test_logic import api_test_logic
+from yaml import load, Loader
+
+import logging 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+tests = api_test_logic
 
 neighbors = static_vals.TEST_NEIGHBORS
 testHash = static_vals.TEST_HASH
 testTrytes = static_vals.TEST_TRYTES
 
 config = {}
-responses = {'getNodeInfo':{},'getNeighbors':{},'getTips':{},'getTrytes':{}}   
+responses = {'getNodeInfo':{},'getNeighbors':{},'getTips':{},'getTransactionsToApprove':{},'getTrytes':{}}   
+
+yamlPaths = ["./tests/features/machine1/output.yml","./tests/features/machine2/output.yml",
+             "./tests/features/machine3/output.yml"]
+
+#Configuration
+@before.all
```

Comment:
It should yes. Since there were so many changes spread out over 2 months to break down and move around there are some instances of copy code in the PR's since parts of the functionality built into one of them is necessary for the testing parts later. It is something I was hoping to discuss with you today